include/fmt/os.h

```diff
@@ -394,7 +394,7 @@ struct ostream_params {
 static constexpr detail::buffer_size buffer_size;
 
 /** A fast output stream which is not thread-safe. */
-class ostream final : private detail::buffer<char> {
+class FMT_API ostream final : private detail::buffer<char> {
```

Comment:
Why move `FMT_API` to class? `grow` is the only non-inline member.

Reply:
The vtable also needs to be exported. Failing to do so, causes linker errors.

After enabling the tests, there seem to be more problems with `-fvisibility=hidden` in shared library builds. It may be good idea to enable the flag in CI builds.